tests/acceptance/features/apiAuthApp/token.feature

```diff
@@ -129,3 +129,12 @@ Feature: create auth-app token
         }
       }
       """
+
+  @env-config
+  Scenario: user deletes auth-app token
```

Comment:
> Can't delete auth-app token using the token received from create request but deleting with the token that we get when listing works.

 It seems multiple ways are present to delete auth-app token. Please specify that in scenario